_docs/templates/_npe2_readers_guide.md.jinja

```diff
@@ -57,7 +57,7 @@ from napari_plugin_engine import napari_hook_implementation
 @napari_hook_implementation
 def napari_get_reader(path: PathOrPaths) -> Optional[ReaderFunction]:
     # If we recognize the format, we return the actual reader function
-    if isinstance(path, str) and path.endswith("*.xyz"):
+    if isinstance(path, str) and path.endswith(".xyz"):
```

Comment:
`str.endswith` just does plain string matching not glob-like matching, so using something like this in a plugin doesn't work (unless the filename actually contains an `*` before the `.xyz` I guess?).